src/stylesheets/settings/_settings-components.scss

```diff
@@ -27,6 +27,7 @@ $theme-alert-bar-width: 1 !default;
 $theme-alert-font-family: "ui" !default;
 $theme-alert-icon-size: 5 !default;
 $theme-alert-padding-x: 2.5 !default;
+$theme-alert-padding-y: 2 !default;
```

Comment:
Add explicit `padding-y` setting instead of calculating it from `padding-x`, which isn't expected.